Documentation has incorrect :system-test:main instructions

I was following the quick start procedure on OS X:
```
$ docker-machine create -d virtualbox --virtualbox-memory 4096 --virtualbox-cpu-count 2 mesos-es
$ eval $(docker-machine env mesos-es)
$ sudo route delete 172.17.0.0/16; sudo route -n add 172.17.0.0/16 $(docker-machine ip mesos-es)
$ ./gradlew build system-test:main
```
The last step fails at 98% with:
```
[INFO] 2016-01-05 22:49:18,845 com.containersol.minimesos.container.AbstractContainer pullImage - Image [jplock/zookeeper:3.4.6] not found. Pulling...
[INFO] 2016-01-05 22:50:10,006 com.containersol.minimesos.container.AbstractContainer pullImage - Image [containersol/mesos-master:0.25.0-0.2.70.ubuntu1404] not found. Pulling...
[INFO] 2016-01-05 22:51:16,397 com.containersol.minimesos.container.AbstractContainer pullImage - Image [containersol/mesos-agent:0.25.0-0.2.70.ubuntu1404] not found. Pulling...
[INFO] 2016-01-05 22:51:29,568 org.apache.mesos.elasticsearch.systemtest.Main main - Starting scheduler
[ERROR] 2016-01-05 22:51:29,869 com.containersol.minimesos.container.AbstractContainer remove - Could not remove container elasticsearch-scheduler_-646294236
com.github.dockerjava.api.NotFoundException: no such id: ""

	at com.github.dockerjava.jaxrs.filter.ResponseStatusExceptionFilter.filter(ResponseStatusExceptionFilter.java:47)
	at org.glassfish.jersey.client.ClientFilteringStages$ResponseFilterStage.apply(ClientFilteringStages.java:134)
	at org.glassfish.jersey.client.ClientFilteringStages$ResponseFilterStage.apply(ClientFilteringStages.java:123)
	at org.glassfish.jersey.process.internal.Stages.process(Stages.java:171)
	at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:251)
	at org.glassfish.jersey.client.JerseyInvocation$1.call(JerseyInvocation.java:667)
	at org.glassfish.jersey.client.JerseyInvocation$1.call(JerseyInvocation.java:664)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:228)
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:424)
	at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:664)
	at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:399)
	at org.glassfish.jersey.client.JerseyInvocation$Builder.delete(JerseyInvocation.java:348)
	at com.github.dockerjava.jaxrs.RemoveContainerCmdExec.execute(RemoveContainerCmdExec.java:27)
	at com.github.dockerjava.jaxrs.RemoveContainerCmdExec.execute(RemoveContainerCmdExec.java:11)
	at com.github.dockerjava.jaxrs.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:24)
	at com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:33)
	at com.github.dockerjava.core.command.RemoveContainerCmdImpl.exec(RemoveContainerCmdImpl.java:78)
	at com.containersol.minimesos.container.AbstractContainer.remove(AbstractContainer.java:114)
	at org.apache.mesos.elasticsearch.systemtest.Main$1.runava:36)
:system-test:main FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':system-test:main'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk1.8.0_51.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 4 mins 48.095 secs
```

--- a/docs/index.md
+++ b/docs/index.md
@@ -329,7 +329,7 @@ $ ./gradlew build buildDockerImage system-test:main
 $ docker-machine create -d virtualbox --virtualbox-memory 4096 --virtualbox-cpu-count 2 mesos-es
 $ eval $(docker-machine env mesos-es)
 $ sudo route delete 172.17.0.0/16; sudo route -n add 172.17.0.0/16 $(docker-machine ip mesos-es)
-$ ./gradlew build system-test:main
+$ ./gradlew buildDockerImage system-test:main
 ```
 
 ### System test

